### PR TITLE
Added ability to use of custom attributes

### PR DESCRIPTION
**Pull request by syberon**
Added ability to use of custom attributes on form elements (for example
'v-model' attribute for Vue). Attributes can be assigned via config
file. Also added ability to use of groups of custom attributes by
defining it via allowed prefixes array.

### CODE REVIEW

File: src/TwbsHelper/Form/View/Helper/FormElement.php [hunk 1]
```diff
@@ -168,6 +168,31 @@ public function render(ElementInterface $oElement)
...
+     *
+     * @param string $name
+     * @param ElementInterface $element
+     * @return string
+     */
+    protected function renderHelper($name, ElementInterface $element)
+    {
+        $helper = $this->getView()->plugin($name);
+
+        if (count($this->options->getValidTagAttributes())) {
```
Comment (neilime) [inline src/TwbsHelper/Form/View/Helper/FormElement.php]: `count` is useless : `if ($this->options->getValidTagAttributes())` is most effective

File: src/TwbsHelper/Form/View/Helper/FormElement.php [hunk 2]
```diff
@@ -168,6 +168,31 @@ public function render(ElementInterface $oElement)
...
+            foreach ($this->options->getValidTagAttributes() as $attribute) {
+                $helper->addValidAttribute($attribute);
+            }
+        }
+
+        if (count($this->options->getValidTagAttributePrefixes())) {
```
Comment (neilime) [inline src/TwbsHelper/Form/View/Helper/FormElement.php]: `count` is useless

**Comment by neilime** [comment]
@syberon can you fix the two review comments. After that, I will merge your PR.

Thank you

**Comment by syberon** [comment]
No problem. I fixed this.

**Pull request merged**